image_provisioner/openshift-templates/pbench/pbench-agent-daemonset.yml

```diff
@@ -0,0 +1,40 @@
+apiVersion: extensions/v1beta1
```

Comment:
This is not how the file structure of ansible playbooks & roles is organized. Please move into the correct directory.